android/src/main/kotlin/com/phyllo/connect/phyllo_connect/PhylloConnectPlugin.kt

```diff
@@ -48,15 +48,42 @@ class PhylloConnectPlugin : FlutterPlugin, MethodCallHandler, EventChannel.Strea
             }
             "initialize" -> {
 
-                Log.d(logTag, call.arguments.toString())
+                val clientDisplayName = call.argument<String?>("clientDisplayName")
+                val userId = call.argument<String?>("userId")
+                val token = call.argument<String?>("token")
+                val environment = call.argument<String?>("environment")
+                val workPlatformId = call.argument<String?>("workPlatformId") ?: ""
+                val singleAccount = call.argument<Boolean?>("singleAccount") ?: false
+                
+                if (clientDisplayName == null) {
+                    showToast("Please pass a valid clientDisplayName.")
+                    result.success(false)
+                }
 
-                val clientDisplayName = call.argument<String>("clientDisplayName") ?: ""
-                val userId = call.argument<String>("userId") ?: ""
-                val token = call.argument<String>("token") ?: ""
-                val environment = call.argument<String>("environment") ?: ""
-                val workPlatformId = call.argument<String>("workPlatformId") ?: ""
+                if (userId == null) {
+                    showToast("Please pass a valid userId.")
+                    result.success(false)
+                }
+
+                if (token == null) {
+                    showToast("Please pass a valid token.")
+                    result.success(false)
+                }
 
-                initialize(clientDisplayName, userId, token, environment, workPlatformId)
+                if (environment == null) {
+                    showToast("Please pass a valid environment.")
+                    result.success(false)
+                }
+
+                initialize(
```

Comment:
This should take a single argument with is a JSON

Reply:
its method for internal manage for bridge mehtod , its should effect anywhere .